api/spec/src/billing/invoice/bill.tsp

```diff
@@ -0,0 +1,407 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace Invoice.Bill;
+
+@summary("Invoice summary represents an extract of the invoice object")
+@friendlyName("BillingInvoiceSummary")
+model InvoiceSummary {
+  ...OpenMeter.InvoiceSummary; // OpenMeter schema extensions
+  type?: InvoiceType;
+
+  @summary("Used as a prefix to group codes.")
+  @visibility("read", "query")
+  series?: CBC.Code;
+
+  @summary("Sequential code used to identify this invoice in tax declarations.")
+  // NOTE: This is required in the original schema, but we can't enforce it here as we will only know this when the
+  // invoice is issued.
+  @visibility("read", "query")
+  code?: CBC.Code;
+
+  @summary("Currency for all invoice totals.")
+  currency?: Currency.Code;
+
+  @summary("Key information regarding previous invoices and potentially details as to why they were corrected.")
+  @visibility("read", "query") // We only populate this for voided invoices
+  preceding?: Org.DocumentRef;
+
+  @summary("Summary of all the invoice totals, including taxes (calculated).")
+  @visibility("read", "query")
+  totals?: Totals;
+
+  @summary("Unstructured information that is relevant to the invoice, such as correction or additional legal details.")
+  @visibility("read", "query")
+  notes?: CBC.Note[];
+
+  @summary("Additional semi-structured data that doesn’t fit into the body of the invoice.")
+  @visibility("read", "query")
+  meta?: CBC.Meta;
+}
+
+namespace OpenMeter {
```

Comment:
do we need namespace here? We are already in Invoice namespace, no?

Reply:
In order for us to have the chance to distinguish between our extensions and the Gobl schema. See [README.md](https://github.com/openmeterio/openmeter/pull/1641/files#diff-44d0a2bc7ca532a2725b5e4284ff129356b058f537ab2d2db519f2b8d3f0e729).

I can drop this part but if we do so, then we will have hard time understanding the diffs between gobl.